### DISCUSSION TITLE
I tried to boot windows 11 from a vhdx file and it went to the startup.nsh

### DISCUSSION THREAD

**Discussion by Vihaan7207** [utmapp/UTM#5069] [UTM Help (macOS)]

**Comment by trey-a-12** [comment]
This solved it for me:

VHDX files have had more issues in my experience. I recommend getting an ARM64 file instead. Your best site for that is [https://uupdump.net/known.php](https://github.com/utmapp/UTM/discussions/url), where you can look for a Windows 11 ARM64 ISO package and convert the download to an ISO. I found it easier to do on a secondary Windows PC, where I downloaded and converted the file to an ISO on the PC, put it in OneDrive on the PC, then downloaded it to my Mac, using that to install instead of the VHDX file.

With the new ISO, now you can make a new VM, setting it to Virtualize Windows, select the ISO, set your settings, go through the normal installer, reboot and detach the ISO, and then go through normal Windows setup. You may have to use the Command Prompt to finish setup, enbling the "I don't have internet" option, which you can look up how to do (I forgot, sorry). After finishing setup, install and run the Windows Guest Tools, and you should be good to go!
